pytube/helpers.py

```diff
@@ -331,5 +325,6 @@ def create_mock_html_json(vid_id) -> Dict[str, Any]:
     logger.info(f'Outputing json.gz file to {gzip_filepath}')
     with gzip.open(gzip_filepath, 'wb') as f:
         f.write(json.dumps(html_data).encode('utf-8'))
+        f.close()
```

Comment:
You most likely do not need to call close() on the file handle if you already use a context manager (the open(...) as f above). The context manager calls it automatically for you.

Reply:
yea i just realized that with already does that. mb